src/lib/core/common-behaviors/color.spec.ts

```diff
@@ -7,13 +7,8 @@ describe('MixinColor', () => {
     const classWithColor = mixinColor(TestClass);
     const instance = new classWithColor();
 
-    expect(instance.color)
-        .toBeFalsy('Expected the mixed-into class to have a color property');
-
-    instance.color = 'accent';
-
-    expect(instance.color)
-        .toBe('accent', 'Expected the mixed-into class to have an updated color property');
+    expect('color' in instance)
+        .toBeTruthy('Expected the mixed-into class to have a color property');
```

Comment:
We are testing if the _property assignment_ works because since we use a `getter/setter` it's not guaranteed that the assigned value becomes the _actual_ value of the getter.

Reply:
As I think about this further, I actually think both test types are necessary and should be split. The original test:

> mixinColor should augment an existing class with a color property

I think is now tested properly, but to your point, I think we need a different test name for the scenario you're describing. I'm open to suggestions on this, as I think: 

> mixinColor should literally set the color value to the value assigned

is an awkward phrasing.

Reply:
I think the test logic is sufficient, but I'm not too sure about the spec name. It feels detailed enough to me, but maybe it depends on personal preference. cc. @jelbourn for feedback.